sdks/android/Makefile

```diff
@@ -160,16 +164,42 @@ package: $(patsubst %,$(APK_DIR)/jni/arm64-v8a/%,$(MONO_LIBS))
 package: $(patsubst %,$(APK_DIR)/jni/x86/%,$(MONO_LIBS))
 package: $(patsubst %,$(APK_DIR)/jni/x86_64/%,$(MONO_LIBS))
 package: $(APK_DIR)/assets/mconfig
-package: $(APK_DIR)/assets/asm/main.exe
 package: $(APK_DIR)/assets/asm/Mono.Android.dll
 package: $(patsubst %,$(APK_DIR)/assets/asm/%,$(BCL_LIBS))
 package: $(patsubst %,$(APK_DIR)/assets/asm/%,$(NUNIT_LIBS))
 package: $(patsubst %,$(APK_DIR)/assets/asm/monodroid_%_test.dll,$(CHECKS))
 
+package-bcl: package
+	rm -rf $(APK_DIR)/bin/AndroidActivity-debug.apk
```

Comment:
Why do we need to remove that? Is it a dependency that is wrong/faulty and make it not rebuild while it should?

Reply:
If you look at lines 19-22, I've made this a conditional dependency now. The debugger test app doesn't have a main.exe file, we only make this for the bcl. We generate this file directly by name in the spot where the compile the bcl (package-bcl).

Reply:
:+1: